src/main/java/vg/civcraft/mc/civmodcore/CivModCorePlugin.java

```diff
@@ -1,28 +1,52 @@
 package vg.civcraft.mc.civmodcore;
 
+import org.bukkit.configuration.serialization.ConfigurationSerialization;
+import org.bukkit.event.HandlerList;
+import vg.civcraft.mc.civmodcore.api.ItemAPI;
+import vg.civcraft.mc.civmodcore.chatDialog.ChatListener;
+import vg.civcraft.mc.civmodcore.chatDialog.DialogManager;
+import vg.civcraft.mc.civmodcore.dao.ManagedDatasource;
+import vg.civcraft.mc.civmodcore.inventorygui.ClickableInventoryListener;
+import vg.civcraft.mc.civmodcore.itemHandling.NiceNames;
 import vg.civcraft.mc.civmodcore.playersettings.gui.ConfigCommand;
+import vg.civcraft.mc.civmodcore.scoreboard.ScoreBoardListener;
 
 public class CivModCorePlugin extends ACivMod {
 
 	private static CivModCorePlugin instance;
 
 	@Override
-	public void onEnable() {
+	public synchronized void onEnable() {
```

Comment:
Why is this synchronized. All calls to onEnable and onDisable will always come from the same thread

Reply:
The original code looks like so

```java
@Override
public void onEnable() {
	initApis(this);
}

private static synchronized void initApis(ACivMod instance) {
	if (!initializedAPIs) {
		// Initialises APIs
	}
}
```
whereby the API initialisation is synchronized. I figured it'd be best to continue that, though I'm not sure if onEnable and onDisable will be synchronized the same way that initApis was, but this is black magic to me, but making something synchronized makes it *more* thread safe, right?

Reply:
No, adding more synchronized introduces the danger of dead locks, you cant just add more of them to make it safer. Just remove all synchronized here, it's not needed.

Reply:
Fair enough